docs/source/histoqc.md

```diff
@@ -0,0 +1,72 @@
+# Instructions to setup and run HistoQC on H&E slides datasets
+
+## Install Openslide
+
+```shell
+sudo apt-get install openslide-tools
+sudo apt-get install python3-openslide
```

Comment:
Does OpenSlide need to be installed at system level? Would it not work inside the conda env like we use? And does the HistoQC env not include OpenSlide?

Reply:
I am not sure why this needs to be installed at the system level, perhaps @kenza-bouzid could reply? Histoqc env includes Openslide.

Reply:
If the virtual env includes it already no need to install it at a system level I believe, this is how we use it within HimlHisto as well!